massa-execution-worker/src/tests/scenarios_mandatories.rs

```diff
@@ -344,7 +449,9 @@ mod tests {
             ..Default::default()
         });
 
-        println!("events: {:?}", events);
+        assert!(events[0]
+            .data
+            .contains("tokens sent to the SC during the call : 10000000000"));
```

Comment:
Should not be hardcoded?

Reply:
Yes nice idea. Should be done now :) Thank you !